sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -26,6 +26,18 @@
 
 #include <azure/core/_az_cfg_prefix.h>
 
+/**
+ * @brief Azure IoT Hub Client method_twin_content_type option value: CBOR.
+ * @note It can be used by wrapping the macro in an #AZ_SPAN_FROM_STR macro as a parameter.
+ */
+#define AZ_IOT_HUB_CLIENT_OPTION_METHOD_TWIN_CONTENT_TYPE_CBOR "application%2Fcbor"
+
+/**
+ * @brief Azure IoT Hub Client method_twin_content_type option value: JSON.
+ * @note It can be used by wrapping the macro in an #AZ_SPAN_FROM_STR macro as a parameter.
+ */
+#define AZ_IOT_HUB_CLIENT_OPTION_METHOD_TWIN_CONTENT_TYPE_JSON "application%2Fjson"
```

Comment:
might be a dumb question, but in what scenario would someone use this (if it defaults to JSON)? I'm guessing this is just for clarity/parity?

Reply:
It is a supported option, so I'd like to include it for completeness and documentation.  Specific use case, maybe a group of devices that some use cbor and some use json, or for some reason need to switch part way in their life. There is a nice consistency to the implementation to have both supported.  But right away, I doubt anyone will use this option for json.